src/ontology/mondo-edit.obo

```diff
@@ -517258,15 +517263,15 @@ xref: GARD:1516 {source="Orphanet:1662"}
 xref: ICD10CM:Q82.8 {source="Orphanet:1662", source="Orphanet:1662/attributed", source="Orphanet:1662/ntbt", source="DOID:0060762"}
```

Comment:
synonyms with exact synonym  from "Orphanet:1662" should probably be reviewed.

Reply:
done